# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libjpeg/boolean-hotfix.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Most sources devince FALSE or TRUE and thus expand the boolean enum to
0 = 0, 1 = 1, or similar constructs.

  - Rene Rebe <rene@exactcode.de>

--- jpeg-9/jmorecfg.h.vanilla	2014-01-04 09:01:48.938763723 +0000
+++ jpeg-9/jmorecfg.h	2014-01-04 09:02:13.982764869 +0000
@@ -252,15 +252,14 @@
  * Defining HAVE_BOOLEAN before including jpeglib.h should make it work.
  */
 
-#ifdef HAVE_BOOLEAN
 #ifndef FALSE			/* in case these macros already exist */
 #define FALSE	0		/* values of boolean */
 #endif
 #ifndef TRUE
 #define TRUE	1
 #endif
-#else
-typedef enum { FALSE = 0, TRUE = 1 } boolean;
+#ifndef HAVE_BOOLEAN
+typedef enum { __FALSE = 0, __TRUE = 1 } boolean;
 #endif
 
 
